crates/fluvio-cli/src/connector/mod.rs

```diff
@@ -318,6 +447,20 @@ fn full_yaml_test() {
             "param_6".to_string(),
             vec!["-10".to_string(), "-10.0".to_string()].into(),
         ),
+        ("aggregate".to_string(), "myaggregate".to_string().into()),
```

Comment:
Part of me thinks that we should have the parameters going to the connectors as `smartmodule-aggregate`, `smartmodule-map`, etc. similar to `producer-compression`, `producer-linger`, etc.

Reply:
I think that in the future we should just use `--smartmodule`. I can change with `--smartmodule-name` if needed but I use --smartmodule in order to be similar to `fluvio consume` flag.

I just keep the old --map, --filter, --aggregate... for backward compatibility but maybe in the future we can just remove those options.

Reply:
ah, yes. If `--smartmodule` is just the name of the smartmodule, that works. I agree that we should keep the `map`, `filter`, `aggregate` for now to maintain backward compatability. Ideally we could have both but if we add a `--smartmodule-map`, etc. it'll break the connector start as it'll be an "unexpected commandline option". It's burdensome that we have to update two repo's at the sametime to change the name. :/